wren-ai-service/tools/dev/.env.example

```diff
@@ -13,7 +13,7 @@ IBIS_SERVER_PORT=8000
 WREN_PRODUCT_VERSION=development
 WREN_ENGINE_VERSION=latest
 WREN_AI_SERVICE_VERSION=latest
-WREN_UI_VERSION=latest
+WREN_UI_VERSION=feedback-loop-1
```

Comment:
IMO, it’s better to keep the example at the latest version. You can specify a particular version in your personal environment file if needed.

Reply:
yes